config/hooks/pre-commit

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+set -e
+
+REPO_ROOT_DIR="$(git rev-parse --show-toplevel)"
+
+files=$((git diff --cached --name-only --diff-filter=ACMR | grep -Ei "\.java$") || true)
+if [ ! -z "${files}" ]; then
+    comma_files=$(echo "$files" | paste -s -d "," -)
+    "${REPO_ROOT_DIR}/gradlew" goJF -DgoogleJavaFormat.include="$comma_files" &>/dev/null
```

Comment:
POSIX sh does not mandate support for `&>`. Replace `&>/dev/null` with POSIX-compliant equivalent `>/dev/null 2>&1`.

Caught by `shellcheck` linter.